### ISSUE TITLE
Cannot install it lack of permissions.

### ISSUE THREAD

**Issue by SirMartin** [frostming/pdm-autoexport#2]
Hi I am trying to install the plugin, but I got this error. Any idea?

```
Installation failed: 
Collecting pdm-autoexport
  Using cached pdm_autoexport-0.1.1-py3-none-any.whl (4.0 kB)
Installing collected packages: pdm-autoexport
ERROR: Could not install packages due to an OSError: [Errno 13] Permission 
denied: '/usr/local/lib/python3.10/dist-packages/pdm_autoexport'
Consider using the `--user` option or check the permissions.
```

**Comment by frostming** [comment]
The error cause is obvious, try raising the permission.

**Comment by SirMartin** [comment]
I don´t have PDM as Sudo level, and I cannot pass the --user, I suppose is part of pip, and PDM doesn´t forward it to pip.

**Comment by frostming** [comment]
How did you install PDM, via any recommended way or system package manager?

**Comment by SirMartin** [comment]
I think I installed it from pip, not sure if time ago was a recommended way, I uninstalled and installed in an actual way, and worked fine with the plugin installation. Thanks.

Another question, if I want to install this plugin also in a docker container in the future, installing PDM through pip could give issues to the plugin installation. Because in the PDM doc, speaking of docker it installs via PIP.

Thanks again for your work and support.
